src/hyperlight_host/src/hypervisor/crashdump.rs

```diff
@@ -2,21 +2,21 @@ use std::io::Write;
 
 use tempfile::NamedTempFile;
 
-use super::Hypervisor;
+use super::hyperlight_vm::HyperlightSandbox;
+use crate::hypervisor::HyperlightVm;
 use crate::{new_error, Result};
 
 /// Dump registers + memory regions + raw memory to a tempfile
-#[cfg(crashdump)]
```

Comment:
if you're wondering why this was removed, it's because this entire crashdump module is already gated on this